File: swarmsolver-frontend/src/app/api/models/task-summary-list-dto.ts
/* tslint:disable */
/* eslint-disable */
import { FilterDto } from '../models/filter-dto';
import { SortingDto } from '../models/sorting-dto';
import { TaskSummaryDto } from '../models/task-summary-dto';
export interface TaskSummaryListDto {
  filtering?: FilterDto;
  sorting?: SortingDto;
  summaries?: Array<TaskSummaryDto>;
  workspace?: string;
}

File: swarmsolver-backend/src/main/java/ai/swarmsolver/backend/app/task/dto/TaskSummaryListDTO.java
package ai.swarmsolver.backend.app.task.dto;

import lombok.*;

import java.util.List;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
@Getter
public class TaskSummaryListDTO {
    private String workspace;
    private List<TaskSummaryDTO> summaries;
    private FilterDTO filtering;
    private SortingDTO sorting;
}
